code/mod_table/mod_table.cpp

```diff
@@ -37,6 +37,7 @@ bool Beams_use_damage_factors = false;
 float Generic_pain_flash_factor = 1.0f;
 float Shield_pain_flash_factor = 0.0f;
 gameversion::version Targetted_version(2, 0, 0, 0); // Defaults to retail
+int MAX_BITMAPS = DEFAULT_MAX_BITMAPS;
```

Comment:
This variable was declared as `exten` in bmpman.h but the actual variable is stored in mod_table.cpp. The extern declaration and variable definition should be in the same header/source pair.

Reply:
I wanted it available to everything that previously had access to it (by importing bmpman.h) without forcing them all to include mod_table.h.

Reply:
Then maybe the variable should be defined in bmpman.cpp?

Reply:
That works.